delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutesSpec.scala

```diff
@@ -134,83 +131,82 @@ class ResourcesRoutesSpec extends BaseRouteSpec with IOFromMap with CatsIOValues
 
   private lazy val routes = routesWithDecodingOption(DecodingOption.Strict)._1
 
-  private val payloadUpdated = payload deepMerge json"""{"name": "Alice", "address": null}"""
-
-  private val payloadUpdatedWithMetdata = payloadWithMetadata deepMerge json"""{"name": "Alice", "address": null}"""
+  private val payloadUpdated             = payload deepMerge json"""{"name": "Alice", "address": null}"""
+  private def payloadUpdated(id: String) = payloadF(id) deepMerge json"""{"name": "Alice", "address": null}"""
 
   private val varyHeader = RawHeader("Vary", "Accept,Accept-Encoding")
 
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    aclCheck.append(AclAddress.Root, reader -> Set(resources.read)).accepted
+    aclCheck.append(AclAddress.Root, writer -> Set(resources.read)).accepted
+    aclCheck.append(AclAddress.Root, writer -> Set(resources.write)).accepted
+  }
```

Comment:
The permissions are set once, making all test independent w.r.t. permissions. Anonymous has no permissions; reader can only read; writer can read and write